cli/tests/coverage/test_cover_branch_pretty.out

```diff
@@ -0,0 +1 @@
+o
```

Comment:
Remove this file, added by accident (correct output file is test_coverage_branch.out).